### PR TITLE
Remove Lottie

### PR DESCRIPTION
**Pull request by yurii-glia**
Replacing operator typing indicator with native CoreAnimation. 
MOB-2267

### CODE REVIEW

**Review by igorkravchenko — APPROVED** [review]
Looks good 👍

**Comment by igorkravchenko** [comment]
Seems like this affects snapshot tests in some way.

**Review by gersonnoboa — APPROVED** [review]

**Review by rasmustautsglia — APPROVED** [review]

**Review by EgorovEI — APPROVED** [review]

File: GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift [hunk 1]
```diff
@@ -1,73 +1,157 @@
...
-    private let animationView = AnimationView()
-    private let kViewSize = CGSize(width: 28, height: 28)
-    private let kLeftInset: CGFloat = 10
-    private let bundleManaging: BundleManaging
+    lazy var dotLeadingView = DotView(
+        color: style.color,
+        duration: animationDuration,
+        beginTime: 0
+    )
+    lazy var dotMiddleView = DotView(
```
Comment (EgorovEI) [inline GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift:11]: Minor. Looks like dot views can be `private`

File: GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift [hunk 2]
```diff
@@ -1,73 +1,157 @@
...
     }
 
     private func updateAccessibility() {
         accessibilityLabel = style.accessibility.label.withOperatorName(accessibilityProperties.operatorName)
     }
 }
 
+extension OperatorTypingIndicatorView {
+    final class DotView: BaseView {
+        let color: UIColor
```
Comment (EgorovEI) [inline GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift:73]: looks like these properties also can be `private`

**Pull request merged**